app/src/main/res/values/attrs.xml

```diff
@@ -63,4 +63,8 @@
         <attr name="type" format="string"/>
     </declare-styleable>
     <attr name="textInputLayoutTheme" format="reference"/>
+
+    <declare-styleable name="CustomTextView">
```

Comment:
1. Change this to an array with the *title* and *subtitle* options.

Reply:
done